test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -198,6 +198,16 @@ class ExportsTest {
     assertEquals(7, bar.y)
   }
 
+  @Test def properties_are_not_enumerable(): Unit = {
+    class Foo {
+      @JSExport
+      var myProp: Int = 1
```

Comment:
Maybe for consistency with the Scala.js-defined JS tests, use a `def` here as well. Though for Exports even `val`s and `var`s are properties, so this does not change anything here.